### PR TITLE
[Feature] Enable Sepolia testnet by default

### PR DESCRIPTION
**Pull request by smohamedjavid**
### Summary

This PR 
- enables Sepolia test network by default
- adds a toggle to switch to the Goerli test network

status-go PR: https://github.com/status-im/status-go/pull/4770

### Testing notes

A complete testing of Wallet is appreciated.

### Platforms

- Android
- iOS

### Steps to test

- Open Status
- Navigate to `Profile > Advanced`
- Enable `Testnet mode` (Make sure the toggle `Enable Goerli as test network` is OFF)
- Navigate to the Wallet tab
- Verify assets from the Sepolia test network are shown
- Enable `Enable Goerli as test network` toggle
- Verify assets from the Goerli test network are shown

status: ready

### CODE REVIEW

File: src/status_im/constants.cljs [hunk 1]
```diff
@@ -415,14 +415,14 @@
...
-(def ^:const optimism-testnet-chain-id 420)
+(def ^:const ethereum-mainnet-chain-id 1)
+(def ^:const ethereum-goerli-chain-id 5)
 (def ^:const ethereum-sepolia-chain-id 11155111)
+(def ^:const arbitrum-mainnet-chain-id 42161)
+(def ^:const arbitrum-goerli-chain-id 421613)
 (def ^:const arbitrum-sepolia-chain-id 421614)
+(def ^:const optimism-mainnet-chain-id 10)
+(def ^:const optimism-goerli-chain-id 420)
 (def ^:const optimism-sepolia-chain-id 11155420)
```
Comment (OmarBasem) [inline src/status_im/constants.cljs:426]: Aren't these changes already being done in another PR?

Comment (smohamedjavid) [reply src/status_im/constants.cljs:426]: Yes, This is much needed for this PR. I have included it. Not sure when the other PR will go through testing. 
We will fast-track this PR to get this merged.

**Review by OmarBasem — APPROVED** [review]

**Comment by VolodLytvynenko** [comment]
hey @smohamedjavid, thank you for PR. no issues from my side. PR is ready to be merged

**Review by J-Son89 — APPROVED** [review]
